pack/side/tmg_encounter.json

```diff
@@ -0,0 +1,1411 @@
+    "faction_code": "neutral",
+    "hidden": true,
+    "illustrator": "Robert Laskey",
+    "is_unique": true,
+    "name": "The Pale Lantern",
+    "pack_code": "tmg",
+    "position": 46,
+    "quantity": 1,
+    "subname": "Beguiling Aura",
+    "text": "[action]: Flip over a spellbound card at your location.\n[action] Exhaust The Pale Lantern: Test [combat] or [agility] (2). If you succeed, place 1 damage on it. Then, if there is 4 damage on it, you break it; add it to the victory display (ignoring its <b>Forced</b> effect) and flip over each spellbound card.\n<b>Forced</b> - When The Pale Lantern would leave play: Flip it over and attach it to its location instead.",
+    "traits": "Item. Relic.",
+    "type_code": "asset",
+    "victory": 1
+  },
+  {
+    "code": "71047",
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 47,
+    "enemy_damage": 1,
+    "enemy_evade": 2,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "flavor": "\"I wasn't expecting this kind of entertainment!\" - Elizabeth Conrad",
+    "health": 3,
+    "illustrator": "??",
+    "name": "Enraged Gug",
+    "pack_code": "tmg",
+    "position": 47,
+    "quantity": 2,
+    "text": "Alert. Hunter.\nHorror dealt by this enemy must be assigned to [[Guest]] assets you control, if able.",
+    "traits": "Monster. Dreamlands. Gug.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "71048",
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 49,
+    "enemy_damage": 1,
+    "enemy_evade": 4,
+    "enemy_fight": 2,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "\"Life imitates Art far more than Art imitates Life.\" - Oscar Wilde",
+    "health": 3,
+    "illustrator": "Alexander Chelyshev",
+    "name": "Abhorrent Moon-Beast",
+    "pack_code": "tmg",
+    "position": 48,
+    "quantity": 2,
+    "text": "Hunter. Retaliate.\nDamage dealt by this enemy must be assigned to [[Guest]] assets you control, if able.",
+    "traits": "Monster. Dreamlands. Servitor.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "71049",
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 51,
+    "enemy_damage": 2,
+    "enemy_evade": 1,
+    "enemy_fight": 1,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "And they whispered also that the rumoured Shantak-birds are no wholesome things... - <i>H.P. Lovecraft</i>, The Dream-Quest of Unknown Kadath",
+    "health": 4,
+    "illustrator": "Alexander Chelyshev",
+    "name": "Savage Shantak",
+    "pack_code": "tmg",
+    "position": 49,
+    "quantity": 1,
+    "text": "Hunter.\nSavage Shantak gets +1 fight and +1 evade for each [[Guest]] asset at its location.",
+    "traits": "Monster. Dreamlands. Shantak.",
+    "type_code": "enemy"
+  },
+  {
+    "code": "71050",
+    "encounter_code": "the_midwinter_gala",
+    "encounter_position": 52,
+    "faction_code": "mythos",
+    "illustrator": "??",
```

Comment:
Could not make this out in the spoilers.

Reply:
Illustrator is `Helge C. Balzer`